schema_utils_test.py

```diff
@@ -830,6 +830,24 @@ def test_is_regex_matched(self):
             'WzEuNjI2NTgxNDQwOTVlKzEyXQ==WzE3NThd',
             r'(exploration|collection)\.\w+\.\w+'))
 
+    def is_gae_search_query_string(self):
```

Comment:
Are you sure that this test is running? (I believe the test functions should start with `test_*`)

Reply:
Thank you, now added.